modules/standard/Reflection.chpl

```diff
@@ -50,18 +50,46 @@ private proc checkQueryT(type t) type {
 proc numFields(type t) param : int
   return __primitive("num fields", checkQueryT(t));
 
+/* Get the name of the ith field in a class or record.
```

Comment:
Should this be rephrased in some way to use `idx` instead of `i`?  Maybe something like `Get the name of the field at `idx` in a class or record`?